ui/components/ui/qr-code-view/index.scss

```diff
@@ -6,29 +6,71 @@
   justify-content: center;
   align-items: center;
 
+  &__wrapper {
+    position: relative;
+    display: inline-block;
+  }
+
+  &__image {
+    position: relative;
+    z-index: 1;
+    border: 1px solid var(--color-border-muted);
+    border-radius: 16px;
+    background-color: white;
+  }
+
+  &__logo {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 2;
+    background-color: white;
+    display: flex;
+    overflow: hidden;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    width: 45px;
+  }
+
+  &__logo img {
+    height: 35px;
+    width: 35px;
+  }
+
   &__message-container > div:first-child {
     @include design-system.Paragraph;
 
     margin-top: 18px;
     color: var(--color-text-muted);
   }
 
+  &__address-segments {
+    word-break: break-all;
+    width: 240px;
+    text-align: center;
+  }
+
+  &__address-inner-segment {
+    display: inline;
+  }
+
+  &__copy-button {
+    cursor: pointer;
+  }
+
+  &__message {
+    @include design-system.H7;
```

Comment:
I realize we need to add some stylelinting to deprecated these but the typography mixins have been deprecated in favor of the text component can we use it here instead?

Reply:
Just removed &__message since it is unused